core/src/main/scala/org/http4s/headers/package.scala

```diff
@@ -48,4 +48,13 @@ package object headers {
             .append(idMostSigBits.toHexString)
         }
     }
+
+  @deprecated("Deprecated in favor of HttpToken", "0.18")
```

Comment:
We haven't conclusively decided on the next version, but I'd call it 0.22 for now.